corehq/apps/custom_data_fields/edit_model.py

```diff
@@ -104,7 +104,7 @@ def get_validator(cls, domain):
 
     @classmethod
     def page_name(cls):
-        return _("Edit {} Fields").format(cls.entity_string)
+        return _("Edit {} Fields").format(unicode(cls.entity_string))
```

Comment:
Just curious about forcing to unicode - would it break without this when we upgrade?